examples/kustomize.jsonnet

```diff
@@ -6,6 +6,7 @@ local kp =
   };
 
 local manifests =
+  //{ ['ksm-autoscaler-' + name]: kp.ksmAutoscaler[name] for name in std.objectFields(kp.ksmAutoscaler) } +
```

Comment:
Can we just add a comment explaining what this enables and when a user would need it?

Reply:
> Can we just add a comment explaining what this enables and when a user would need it?

Sure